Mathlib/Algebra/Order/AbsoluteValue/Equivalence.lean

```diff
@@ -0,0 +1,53 @@
+/-
+Copyright (c) 2025 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import Mathlib.Analysis.SpecialFunctions.Pow.Real
+
+/-!
+# Equivalence of real-valued absolute values
+
+Two absolute values `v₁, v₂ : AbsoluteValue R ℝ` are *equivalent* if there exists a
+positive real number `c` such that `v₁ x ^ c = v₂ x` for all `x : R`.
+-/
+
+namespace AbsoluteValue
+
+variable {R : Type*} [Semiring R]
+
+/-- Two absolute values `f, g` on `R` with values in `ℝ` are *equivalent* if there exists
+a positive real constant `c` such that for all `x : R`, `(f x)^c = g x`. -/
+def Equiv (f g : AbsoluteValue R ℝ) : Prop :=
```

Comment:
Also, should this maybe be turned into a `Setoid` instance? To provide nice notation?

Reply:
Good point!